mtlpyrelease/release.py

```diff
@@ -118,6 +117,30 @@ def generate_release_names(num, names, translate=True, show_excluded=False):
 
     return fr_names, en_names
 
+
+def event_num_offsets(num):
+    ''' Return two offsets to define how the words of event names are selected. 
+    
+    Every 26 events, a new series is generated. Inside a series, the 
+    progression of letters from one event to the next is obvious.
+    
+    This system was is not historically compatible with the events up to MP-95. 
+    It was first used at MP-96 and could extend to infinity if needed. '''
+    old_state = getstate()
+    series = 6 + (num - 96) // 26
+    event_offset = (num - 96) % 26
+    
+    # going deterministic for a moment
+    seed(f"This is a beatiful day to hack. Series: {series}")  
+
+    adj_offset = (randrange(26) + event_offset * choice([-1, 1])) % 26
+    nout_offset = (randrange(26) + event_offset * choice([-1, 1])) % 26
+
+    setstate(old_state)  # back to chaos!
```

Comment:
Alternatively, we could use an instance of the Random class and leave the global module state alone.